app/authentication/authenticator.py

```diff
@@ -22,27 +24,31 @@
 
 
 @login_manager.user_loader
-def user_loader(user_id):
+def user_loader(user_id: str) -> Union[str, None]:
```

Comment:
If something can be `None` and one other type, then we should use `Optional` so `Optional[str]`.
```suggestion
def user_loader(user_id: str) -> Optional[str]:
```